src/chainstate/coordinator/comm.rs

```diff
@@ -0,0 +1,243 @@
+use std::{
+    thread, process
+};
+use std::time::{
+    Duration, Instant
+};
+use std::sync::{
+    Arc, RwLock,
+    atomic::{Ordering, AtomicU64}
+};
+
+use crossbeam_channel::{bounded, Sender, Receiver, TrySendError};
+
+
+/// Trait for use by the ChainsCoordinator
+/// 
+pub trait CoordinatorNotices {
+    fn notify_stacks_block_processed(&mut self);
+    fn notify_sortition_processed(&mut self);
+}
+
+pub struct ArcCounterCoordinatorNotices {
```

Comment:
I'm not sure what the purpose of this struct is for?  Is this just for prometheus reporting?

Reply:
It's so that a thread can wait for either a stacks block or a sortition to increment -- this is mostly necessary for the neon_node, whose "run loop" requires some synchronicity with both stacks block processing and sortitions.

Reply:
Gotcha.